src/metabase/util/stats.clj

```diff
@@ -0,0 +1,332 @@
+(ns metabase.util.stats
+  "Functions which summarize the usage of an instance"
+  (:require [clojure.tools.logging :as log]
+            [clj-http.client :as client]
+            (metabase [config :as config]
+                      [db :as db])
+            [metabase.public-settings :as settings]
+            (metabase.models [field :as field]
+                             [table :as table]
+                             [setting :as setting])           
+            [metabase.util :as u]))
+
+(def ^:private ^:const ^String metabase-usage-url "https://kqatai1z3c.execute-api.us-east-1.amazonaws.com/prod/ServerStatsCollector")
+
+(def ^:private ^Integer anonymous-id
```

Comment:
Actually on second thought maybe this shouldn't be `^:const` because you want it calculated on launch time (as opposed to when the JAR is compiled)?

It's worth adding a note to self about not making it const for that reason some some dummy like myself doesn't go and try to make it const